ruma-events/src/room/redaction.rs

```diff
@@ -64,6 +64,12 @@ pub struct RedactionEventContent {
     pub reason: Option<String>,
 }
 
+impl ruma_events::RoomEventContent for RedactionEventContent {}
+
+impl ruma_events::MessageEventContent for RedactionEventContent {}
+
+impl ruma_events::StateEventContent for RedactionEventContent {}
```

Comment:
Nothing except `CustomEventContent` should ever implement both `MessageEventContent` and `StateEventContent`. `m.room.redaction` is a message event, so please remove this.

Reply:
Should this be removed also https://github.com/ruma/ruma/blob/master/ruma-events/src/enums.rs#L69 ?